src/app/grants/page.tsx

```diff
@@ -0,0 +1,194 @@
+import ClaimButton from '@/components/ClaimButton';
+import { Card, CardContent } from '@/components/ui/card';
+import { ScrollArea } from '@/components/ui/scroll-area';
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from '@/components/ui/select';
+import { Separator } from '@/components/ui/separator';
+import { ArrowUpDown, ArrowUpRight, Hexagon, Search } from 'lucide-react';
+import Link from 'next/link';
+
+const grants = [
```

Comment:
please prepare a context for grants and expose this array of grants through context. we will need this info throughout the rest of the app and once we have real data, we will be able to fetch and expose it from context! it can be called `<GrantsContext />`